tools/salmon/salmonquant.xml

```diff
@@ -0,0 +1,167 @@
+<tool id="salmon" name="Salmon quant" version="@VERSION@">
```

Comment:
this tool should overwrite this here: https://github.com/astrovsky01/galaxytools/blob/master/tools/salmon/salmon.xml

Reply:
Removed salmon.xml